src/category_theory/abelian/subobject.lean

```diff
@@ -0,0 +1,57 @@
+/-
+Copyright (c) 2022 Markus Himmel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Markus Himmel
+-/
+import category_theory.abelian.opposite
+import category_theory.subobject.limits
+
+/-!
+# Equivalence between subobjects and quotients in an abelian category
+
+-/
+
+open category_theory category_theory.limits opposite
+
+universes v u
+
+noncomputable theory
+
+namespace category_theory.abelian
+variables {C : Type u} [category.{v} C]
+
+/-- In an abelian category, the subobjects and quotient objects of an object `X` are
+    order-isomorphic via taking kernels and cokernels. -/
```

Comment:
```suggestion
    order-isomorphic via taking kernels and cokernels.
    Implemented here using subobjects in the opposite category,
    since mathlib does not have a notion of quotient objects at the time of writing. -/
```